### PR TITLE
Us1006

### PR DESCRIPTION
**Pull request by minhuyyyy**

### CODE REVIEW

File: src/components/modals/desktop-filter-dropdown.tsx [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+        <div className='w-full h-full overflow-y-scroll '>
+            {/* <FilterComponent
+                key={1}
+                filterName='Mileage'
+                filterValues={}
+            /> */}
+            <div className='divider'>
+                <details id='bodies'>
+                    <summary>
+                        <a
```
Comment (ttnguyen115) [inline src/components/modals/desktop-filter-dropdown.tsx]: anchor elements display multiple times, can split into smaller component

File: src/components/modals/desktop-filter-dropdown.tsx
Comment (ttnguyen115) [inline src/components/modals/desktop-filter-dropdown.tsx:1]: can split into smaller components for more readable

File: src/components/shared/filters/filter-component.tsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
+import { Filter } from '@contracts/types/filter';
+import { handleInputChange } from '@utils/input-utils';
+import React from 'react';
+
+function FilterInput({ filterValues }: { filterValues: Filter }) {
+    return (
+        <>
+            <div className='flex flex-col '>
+                {filterValues.value.map((value, index) => (
+                    <>
```
[the remark this answers is not in the document]
Comment (minhuyyyy) [reply src/components/shared/filters/filter-component.tsx]: updated in pr #3

File: src/components/modals/desktop-filter-dropdown.tsx (hunk 1 above)
Comment (minhuyyyy) [reply src/components/modals/desktop-filter-dropdown.tsx]: updated in pr #3

**Pull request merged**